pages/index.js

```diff
@@ -12,6 +12,25 @@ export default function Home() {
     setEntries([...entries, newEntry]);
   }
 
+  function onUpdateEntry(editedEntry) {
```

Comment:
gemäß Konvention würde ich die Funktion handleUpdateEntry nennen (auch in Zeile 46)